main.cc

```diff
@@ -1696,6 +1696,24 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 start_cql(cql_maintenance_server_ctl, stop_maintenance_cql, "maintenance native server");
             }
 
+            snapshot_ctl.start(std::ref(db)).get();
+            auto stop_snapshot_ctl = defer_verbose_shutdown("snapshots", [&snapshot_ctl] {
+                snapshot_ctl.stop().get();
+            });
+
+            api::set_server_snapshot(ctx, snapshot_ctl).get();
+            auto stop_api_snapshots = defer_verbose_shutdown("snapshots API", [&ctx] {
+                api::unset_server_snapshot(ctx).get();
+            });
+
+            api::set_server_tasks_compaction_module(ctx, ss, snapshot_ctl).get();
+            auto stop_tasks_api = defer_verbose_shutdown("tasks API", [&ctx] {
+                api::unset_server_tasks_compaction_module(ctx).get();
+            });
+
+            //FIXME: discarded future
+            (void)api::set_server_cache(ctx);
+
```

Comment:
`nodetool compact`, `nodetool scrub` and a lot of other important `nodetool` commands are served by the `storage_service` API routes, which I don't see in the above list. Unfortunately, said routes also contains endpoints, which involve cluster-wide operation. We have two choices here:
* Split the `api/storage_service.cc` into node-local and cluster-wide operations, and only enable here the node-local part.
* Just make sure we raise an error if user attempts to invoke a cluster-wide operation.